packages/cashier/src/stores/on-ramp-store.js

```diff
@@ -60,6 +60,7 @@ export default class OnRampStore extends BaseStore {
         });
     }
 
+    /** @deprecated Use `useOnrampVisible` from `@deriv/hooks` package instead. */
     get is_onramp_tab_visible() {
```

Comment:
The `is_onramp_tab_visible` method is still used in the cashier `GeneralStore`, Since we can’t use hooks in stores we need to keep this method until we refactor the `GeneralStore` logic.